unittests/InstructionCountCI/Primary.json

```diff
@@ -4490,21 +4474,23 @@
       ]
     },
     "clc": {
-      "ExpectedInstructionCount": 3,
+      "ExpectedInstructionCount": 4,
       "Comment": "0xf8",
       "ExpectedArm64ASM": [
-        "mrs x20, nzcv",
-        "orr w20, w20, #0x20000000",
-        "msr nzcv, x20"
+        "mov w20, #0x1",
+        "mrs x21, nzcv",
+        "bfi w21, w20, #29, #1",
```

Comment:
What happened here to clc/stc?

Reply:
this is the non-flagm path. speeding it up would add overhead to the flagm path and in terms of actual uops this isn't worse. so I accepted the bandage hit on v8.0 in exchange for not needing Even More Code of dubious value.